draft-irtf-cfrg-hpke.md

```diff
@@ -1207,6 +1207,43 @@ across all labels in this document.
 The `0xFFFF` AEAD ID is reserved for applications which only use the Export
 interface; see {{hpke-export}} for more details.
 
+# API Considerations {#api-considerations}
+
+The high-level HPKE APIs specified in this document are all fallible.
+For example, `Decap()` can fail if the encapsulated key `enc` is invalid,
+and `Open()` may fail if ciphertext decryption fails. The explicit errors
+generated throughout this specification, along with the conditions that
+lead to each error, are as follows:
```

Comment:
`VerifyPSKInputs` raises exceptions instead of errors. I'm not sure if that's on purpose. They should probably become errors as well (maybe `ValidationError`?) and get mentioned here.

Reply:
I think those are best kept as exceptions since they're due to programmer error.